index.js

```diff
@@ -302,6 +302,19 @@ class HyperBee {
     })
   }
 
+  retrieveMetadata () {
```

Comment:
can we call this getMetadata ?

Reply:
sure, my only concern there was that it might cause confusion because of the `metadata` field